src/font/fontkit.rs

```diff
@@ -66,6 +66,13 @@ impl FontBackend for Font {
                 },
             );
 
+        let color = match color {
+            FontColor::SingleColor(color) => color,
+            // Take just 1st color in vector, fontkit have a lot of problems in rasterize, so it is not a big deal
+            // May be delete this font?
+            FontColor::MultipleColor(ref colors) => colors[0],
+        };
```

Comment:
Yes. That's why I wrote a comment there. The rendering is already broken and looks much worse compared to fdue.
So I suggest just removing it rather than trying to fix it.

Reply:
As far as I remember some fonts were broken with fdue backend, that's why fontkit still exists. I'll try to recheck that one today, perhaps some of the recent fixes resolve those cases.

Reply:
Probably I've found some fonts (like `Nimbus Mono PS`) but those are still rendered poorly with fontkit anyway, so I guess I'll remove fontkit completely and finally make a release